.github/workflows/deploy.yml

```diff
@@ -37,60 +37,36 @@ jobs:
       run: |
         cargo build --release
 
-    - name: Run cargo-c
+    - name: Strip rav1e binary
       run: |
-        cargo cinstall --release --destdir="C:\"
+        strip target/release/rav1e.exe
 
-    - name: Create zip
+    - name: Run cargo-c
       run: |
-        $RAV1E_PATH="$Env:GITHUB_WORKSPACE\target\release"
-        7z a rav1e.zip "Cargo.lock" "$RAV1E_PATH\rav1e.exe" "C:\usr\local"
-
-    - name: Upload binaries
-      uses: actions/upload-artifact@v2
-      with:
-        name: rav1e-binaries
-        path: rav1e.zip
-
-  deploy:
-
-    needs: create-binaries
-
-    env:
-      FILENAME: rav1e-windows-sdk
-
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v2
-
-    - name: Download the zip
-      uses: actions/download-artifact@v2
+        cargo cinstall --release --destdir="C:\"
 
-    - name: Unzip rav1e Windows binaries
+    - name: Rename cargo-c folder
       run: |
-        unzip rav1e-binaries/rav1e.zip -d rav1e-binaries
-        mv rav1e-binaries/Cargo.lock .
+        Rename-Item "C:\usr\local" "C:\usr\rav1e-windows-sdk"
 
-    - name: Handle binaries
+    - name: Get the version
+      if: startsWith(github.ref, 'refs/tags/v')
+      shell: bash
+      id: tagName
       run: |
-        mv rav1e-binaries/local rav1e-binaries/$FILENAME
-        strip rav1e-binaries/rav1e.exe
+        VERSION=$(cargo pkgid | cut -d# -f2 | cut -d: -f2)
```

Comment:
I think we need to use https://github.com/marketplace/actions/get-tag-name#get-tag-name instead, `cut` doesn't seem to work on Windows, as I learned from oxipng.

Reply:
Yep, you're right, it shouldn't work on Windows, but if you set `bash: shell`, the bash shell included with `Git` for `Windows` is used. I tested that [here](https://github.com/Luni-4/rav1e/runs/665110574?check_suite_focus=true#step:10:2) and it works.